emperor/support_files/js/model.js

```diff
@@ -5,37 +5,49 @@ define([
 function($, _) {
   /**
    *
-   * @name Plottable
+   * @class Plottable
    *
-   * @class Represents a sample and the associated metadata in the ordination
-   * space.
+   * Represents a sample and the associated metadata in the ordination space.
    *
-   */
-
-
-  /**
-   *
-   * @name Plottable
+   * @param {string} name A string indicating the name of the sample.
+   * @param {string[]} metadata An Array of strings with the metadata values.
+   * @param {float[]} coordinates An Array of floats indicating the position in
+   * space where this sample is located.
+   * @param {integer} [idx=-1] An *optional* integer representing the index
```

Comment:
Here and below - don't need to specify optional since the brackets do that for you.